selfdrive/test/test_models.py

```diff
@@ -22,6 +22,7 @@
 
 # TODO: get updated routes for these cars
 ignore_can_valid = [
+  "AUDI A3",
```

Comment:
this shouldn’t be necessary, what’s failing?

Reply:
nothing i am aware of, just assumed it was needed because the golf is listed here